roles/matrix-postgres/tasks/main.yml

```diff
@@ -26,6 +26,12 @@
   tags:
     - import-synapse-sqlite-db
 
+# Perhaps we need a new variable here, instead of `run_postgres_import_sqlite_db`.
+- import_tasks: "{{ role_path }}/tasks/import_generic_sqlite_db.yml"
```

Comment:
with a `vars: { ... }` block, we should be able to dynamically execute this in every role for auto-migration